go.mod

```diff
@@ -2,6 +2,7 @@ module github.com/spf13/afero
 
 require (
 	github.com/pkg/sftp v1.10.1
+	github.com/stretchr/testify v1.4.0
```

Comment:
I would prefer this to either be used across all unit tests or not at all. Given the scope of your PR, I would remove it for now (feel free to open an issue to track updating all unit tests to use testify).

Reply:
No worries, removed the use of testify and just relying on `t.Fatal(...)` for now.